src/base.ts

```diff
@@ -47,6 +48,39 @@ export abstract class BitArray {
   public abstract getBit(index: number): boolean;
   public abstract setBit(index: number, value: boolean): void;
   public abstract toBitfield(): Uint8Array;
+
+  public or(bitArray: BitArray): BitArray {
+    for(let i = 0; i < this.bitLength; i++) {
+      this.setBit(i, this.getBit(i) || bitArray.getBit(i));
```

Comment:
Thoughts on going byte by byte until the last byte in `this.byteArray`, then going bit by bit?

Reply:
If I assert that they are same length, I can go byte by byte everything?